### PR TITLE
[chore] Prepare release 0.86.0

### LINKED ISSUE #27199: prepare release fails for v0.86.0
### Component(s)

_No response_

### Describe the issue you're reporting

Prepare release automation fails for v0.86.0 because of the jaeger exporter which is no longer published with the new version, but is still included in the testbed binary compilation. See the error message here: 

https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/6307699974/job/17124721508#step:5:601

### PR DESCRIPTION
**Pull request by codeboten**
The following commands were run to prepare this release:
- make chlog-update VERSION=v0.86.0
- sed -i.bak s/0.85.0/0.86.0/g versions.yaml
- make multimod-prerelease
- make multimod-sync

Fixes #27199

### CODE REVIEW

File: cmd/oteltestbedcol/builder-config.yaml.bak
Comment (TylerHelmuth) [inline cmd/oteltestbedcol/builder-config.yaml.bak:1]: Do we need this file?

File: cmd/oteltestbedcol/builder-config.yaml [hunk 1]
```diff
@@ -2,50 +2,50 @@ dist:
...
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/extension/storage v0.86.0
     import: github.com/open-telemetry/opentelemetry-collector-contrib/extension/storage/filestorage
 
 exporters:
   - gomod: go.opentelemetry.io/collector/exporter/loggingexporter v0.86.0
   - gomod: go.opentelemetry.io/collector/exporter/otlpexporter v0.86.0
   - gomod: go.opentelemetry.io/collector/exporter/otlphttpexporter v0.86.0
-  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/carbonexporter v0.85.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/carbonexporter v0.86.0
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/jaegerexporter v0.85.0
```
[the remark this answers is not in the document]
Comment (codeboten) [reply cmd/oteltestbedcol/builder-config.yaml:20]: the testbed still uses it to test the jaeger receiver. i would be happy with an alternative if you have an idea

[the remark this answers is not in the document]
Comment (songy23) [reply cmd/oteltestbedcol/builder-config.yaml:20]: I'd vote for creating an issue and moving forward with release as-is.

**Review by songy23 — APPROVED** [review]

File: cmd/oteltestbedcol/builder-config.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (TylerHelmuth) [reply cmd/oteltestbedcol/builder-config.yaml:20]: > the testbed still uses it to test the jaeger receiver. i would be happy with an alternative if you have an idea

Thats a genius yet annoying idea.

I don't want to block the release for this, but I think we should change how this works. I'm good with an issue.

File: cmd/oteltestbedcol/builder-config.yaml.bak
Comment (codeboten) [reply cmd/oteltestbedcol/builder-config.yaml.bak:1]: we do not.  removed

**Review by mx-psi — APPROVED** [review]

**Review by TylerHelmuth — APPROVED** [review]

**Comment by dmitryax** [comment]
Sorry, I didn't notice this PR and merged https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/27185

**Pull request merged**